app/pages/[page].js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { applySession } from "next-session";
+import { useRouter } from "next/router";
+import { Forms, getHandler } from "../form-schema";
+import Button from "@button-inc/bcgov-theme/Button";
+
+export default function home({ formIndex, formData, validPage, prevPageUrl }) {
```

Comment:
The `prevPageUrl` isn't documented, but I added it to show that you can use this prop to handle backward navigation. If it is equal to `-1`, that means you are at the first page